todos/todo_item.py

```diff
@@ -0,0 +1,15 @@
+class ToDoItem:
+    def __init__(self, index: int, name: str, status: str):
+        self.id = index
+        self.name = name
+        self.status = status
+
+    def has_status(self, status: str):
+        return status == 'all' or self.status == status
```

Comment:
Why not use an actual enum for the status? (https://docs.python.org/3/library/enum.html)